website/docs/d/oauth_token.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_oauth_token"
+sidebar_current: "docs-tfe-datasource-oauth-token"
+description: |-
+  Use this datasource to get the ID of the OAuth token of an assocaited VCS configuration.
+---
+
+# tfe_oauth_token
+
+Use this datasource to get the ID of the OAuth token of an assocaited VCS
+configuration. This token is used to identify which VCS connection to use.
+
+### Example Usage
+
+```hcl
+data "tfe_oauth_token" "oauth_token" {
+  template_filter = "featured"
+}
+```
+
+### Argument Reference
+
+* `template_filter` - (Required) The template filter. Possible values are `featured`, `self`, `selfexecutable`, `sharedexecutable`, `executable` and `community` (see the Cloudstack API *listTemplate* command documentation).
```

Comment:
can we link to the Cloudstack API docs?

Reply:
As above... This file wasn't suppose to get merged. Will delete it from the commit.